runtime/locale/en/ctrl_inventory.hcl

```diff
@@ -202,12 +202,7 @@ locale {
             }
 
             take_ally {
-                refuse_dialog {
-                    _0 = "\"No!\""
-                    _1 = "\"It's mine.\""
-                    _2 = "\"Get off!\""
-                    _3 = "\"Never.\""
-                }
+                refuse_dialog = "\"It's mine.\""
```

Comment:
Is this the correct translation? Before there used to be four items.

Reply:
2nd-4th items were defined, but unused.